packages/drivers/debugger/src/fluidDebuggerController.ts

```diff
@@ -244,19 +244,17 @@ export class DebugReplayController extends ReplayController implements IDebugger
         return this.shouldUseController;
     }
 
+    /**
+     *
+     * @deprecated - only here for back compat, will be removed after release
+     */
     public async read(blobId: string): Promise<string> {
-        if (this.storage !== undefined) {
-            return this.storage.read(blobId);
-        }
-        throw new Error("Reading blob before storage is setup properly");
+            return bufferToBase64(await this.readBlob(blobId));
```

Comment:
fix spacing. Also, isn't the await here changing to read from storage to be synchronous now?